### ISSUE TITLE
Address bar suggestion bug

### ISSUE THREAD

**Issue by 3oheicrw** [snaildos/Fifo-Browser#109]
Instead of the site name, e.g: youtube.com it suggests only a random query like this: https://www.youtube.com/results?search_query=wwe+highlights. I have did many searches on youtube so many queries like this should be recorded. For unknown reasons it only picked that particular query and only suggest it when I type youtube. Turn off search and site suggestions doesn't help. Strangely enough, this query is not on the newtab page too. The newtab page recorded another query and youtube.com itself. Even more weirded, as I have "Clear all history!" but this query is stored somewhere and not cleared at all so Fifo keeps suggesting it to me.

**Comment by snaildos** [comment]
Closing this since it relates to #108

**Comment by snaildos** [comment]
Yeah, I'm not seeing how this issue is really an issue. Regarding the clear all history, that has been fixed, but please note, for your newtab page, you need to restart Fifo to regenerate a fresh newtab page.
Hopefully this clarifies it.
